assets/src/edit-story/components/dropzone/dropZoneProvider.js

```diff
@@ -114,19 +118,47 @@ function DropZoneProvider({ children }) {
     }
   };
 
+  const onDragEnter = (evt) => {
+    disableDefaults(evt);
+    evt.dataTransfer.effectAllowed = 'copy';
+    setIsDragging(true);
+  };
+
+  const onDragLeave = (evt) => {
+    disableDefaults(evt);
+    if (ref.current) {
+      const rect = ref.current.getBoundingClientRect();
+      if (
+        evt.clientY < rect.top ||
+        evt.clientY >= rect.bottom ||
+        evt.clientX < rect.left ||
+        evt.clientX >= rect.right
+      ) {
+        setIsDragging(false);
```

Comment:
`onDragLeave` also wouldn't be necessary if using the `hoveredDropZone` -- this calculation is already done, per dropzone basis.

Reply:
Comment - https://github.com/google/web-stories-wp/pull/376/files#r385023762